docs/basic-component.md

```diff
@@ -4,20 +4,24 @@ Recipe for creating a basic component.
 
 - [Intro](#intro)
 - [Component structure](#component-structure)
+  - [Documentation](#documentation)
   - [Subcomponents](#subcomponents)
   - [Hooks and Utils](#hooks-and-utils)
   - [index.ts](#indexts-1)
   - [Component.tsx](#componenttsx)
   - [Styles](#styles)
   - [Tests](#tests)
+  - [Types](#types)
 - [Component.tsx template](#componenttsx-template)
   - [Props](#props)
   - [The component](#the-component)
   - [Display name](#display-name)
 
 ## Intro
 
-To generate a new component and package, new component within an existing package, or a subcomponent,
+All components are to be added to `packages/components`.
```

Comment:
Would be good to include a definition of the `future` components here since the plop prompts mention it

Reply:
I was afraid you'd ask 😛 Care to help me write something?

Reply:
```suggestion
There are 2 types of components that live in `@kaizen/components`, `src` and `future`, both are **production safe**

### src components
These are components which are essentially the "main" branch of components and only receive non-breaking changes. **Any brand new components should go here.**

### future components
Components which have an existing `src` component equivalent, but have any breaking changes to their API. Eventually `future` components will replace their `src` equivalent in the next major release.
```